code/scripts/templates/testmail.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+echo -e "TORQMONIT\r\rThis is an automated message\r\rProcess SERVICE is down"|mail.mailutils -s \"SUBJECT\" rdanutalexandrugmail.com
```

Comment:
I assume this file should be removed? Doesn't seem like something that should be merged to master

Reply:
This is my attempt to use an external e-mail to generate the alerts. Not very successful so far.